DeveloperGame/DeveloperGame/TrueOrFalseQuestion.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ComputerGeekGame
+{
+    public class TrueOrFalseQuestion : Question
+    {
+        private bool _correctAnswer;
+
+        public TrueOrFalseQuestion(string question, bool correctAnswer) : base(question)
+        {
+            _correctAnswer = correctAnswer;
+        }
+
+        public override void PrintQuestion()
+        {
+            base.PrintQuestion();
+
+            Console.Write("\n\tTrue or False: ");
+        }
+    }
```

Comment:
You will probably need to add a method for checking if the answer is correct. 

What do you think is the best approach?

Reply:
Yes, I think so too. I was trying to work on the Play method and realized that it is kind of awkward to do the checking there. I am going to add an IsTheAnswerCorrect method to these puppies and send it your way.

Reply:
Yep, you have great instincts! 😉

Reply:
Ok, just pushed a commit.